jina/serve/runtimes/worker/__init__.py

```diff
@@ -32,7 +33,29 @@ def __init__(
         """
         super().__init__(args, cancel_event, **kwargs)
 
+        # Keep this initialization order, otherwise readiness check is not valid
+        self._data_request_handler = DataRequestHandler(
```

Comment:
Review this part, as this has changed in a recent commit to master, I guess a bad merge happened here